addons/l10n_es/data/account_tax_data.xml

```diff
@@ -1789,6 +1996,66 @@
             }),
         ]"/>
     </record>
+    <record id="account_tax_template_s_iva0_a" model="account.tax.template">
```

Comment:
I guess the equivalent for 14, 15 is

https://github.com/odoo/odoo/blob/15.0/addons/l10n_es/data/account_tax_data.xml#L2575

```
    <record id="account_tax_template_s_iva0b" model="account.tax.template">
        <field name="type_tax_use">sale</field>
        <field name="name">IVA 0% (Bienes)</field>
        <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
        <field name="amount" eval="0"/>
        <field name="amount_type">percent</field>
        <field name="tax_group_id" ref="tax_group_iva_0"/>
        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
```

Reply:
@rafaelbn Correct for the 0% food taxes, yes

Reply:
Oof, you have changed the XML-IDs and names we put in this PR, and it was for a good reason. Now everything is getting complicated...

Reply:
Exactly, `account_tax_template_s_iva0_a` vs `account_tax_template_s_iva0b` and so on , that why my comment.